frontend/src/pages/account-settings/components/login/components/create-password-form/create-password-form.tsx

```diff
@@ -35,34 +44,62 @@ const CreatePasswordForm: React.FC<Properties> = ({
     [handleSubmit, onSubmit],
   );
 
+  const [showPassword, toggleShowPassword] = useToggle(false);
+  const [showNewPassword, toggleShowNewPassword] = useToggle(false);
+  const [showRepeatPassword, toggleShowRepeatPassword] = useToggle(false);
+
   return (
     <Modal isOpen={isOpen} onClose={onCloseModal}>
-      <div className={getValidClassNames(styles['form-wrapper'], className)}>
+      <div className={getValidClassNames(styles[inputWrapper], className)}>
```

Comment:
Let's get rid of constants for class names, cause in some places you're using it and in other places do not use,, and when I am looking on it - I need to search for a place where these constants are defined, and try to understand on which styles file in relies
```suggestion
      <div className={getValidClassNames(styles['input-wrapper'], className)}>
```

Reply:
lint error in that case

Reply:
share the screen with error please

Reply:
![image](https://user-images.githubusercontent.com/87016333/229342525-4914c194-273d-4b2e-a25e-726adacc7be9.png)

Reply:
just give a different class names

Reply:
even if it shares the same styles?